### ISSUE TITLE
LESS / CSS selector autocompletion not working.

### ISSUE THREAD

**Issue by rm-bergmann** [Komodo/KomodoEdit#3377]
### Short Summary
I have browsed around the issue/bug list and allthough I have found a few autocompletion bug reports, I have not yet found any that mention this specific bug. Apologies if this is a duplicate. 

LESS Autocompletion / CSS selector autocompletion is  not working anymore. This worked well in Komodo IDE 10. 

With LESS I use the namespace feature a lot. Once a namespace has been declared it used to appear in the autocompletion list. To be more specific, if you are not familiar with namespacing in LESS, it is like having methods inside a class.

### Steps to Reproduce
1. Create a less file called display.less. 
2. Declare a name space and some mixins inside of it:

#display {

  .block(@width: 100px; @height: 100px; @col: #000) {
    width: @width;
    height: @height;
    background-colour: @col;
    display: block;
  }

  .circle() {
    #display.block();   // Call the block method / mixin
    border-radius: 50%;
  }

}

// Should be in a different file, like 'style.less' but doesn't need to be for the reproduction steps. To test if it works in different files remember to @import './display'

span {
  #display.circle(); // Call the circle method / mixin
}

### Expected results

Everytime I type '#' the autocomplete should come up with '#display'. If I hit enter and then type '.' when I am expecting to see '.block' and '.circle' in autocomplete.

### Actual results

No autocomplete list after typing '#' or '.'

### Platform Information

*Komodo IDE 11.0.1* 
*Operating System (and version)?* Windows 10 Latest build

Final thing to note:
If a namespace is declared in it's own file (which is how it should be structured) and the mixin method is called in a different style the autocomplete would not pick it up the first time. But when you call the mixin a second time then it would appear in the list

**Comment by Naatan** [comment]
This is simply not implemented at the moment. Our LESS/SCSS CodeIntel support is very basic.

**Comment by rm-bergmann** [comment]
Okay, thanks. But how did it work in 10 and not work in 11 (Just wondering)?

**Comment by Naatan** [comment]
Are you certain this worked in 10? That doesn't sound right to me.

@mitchell-as ping.

**Comment by rm-bergmann** [comment]
Yeah, 100% certain as I used it everyday. It was like a hidden feature I discovered. Then since updating to 11 it dissapeared and I'm really missing it a lot. It saved a lot of typing.

**Comment by rm-bergmann** [comment]
Just want to point something out here. This is not LESS / SCSS specific, the reason it worked with LESS is because LESS uses CSS syntax. So the codeintel was picking up all CSS classes and ID's used in the file. That's my understanding of it anyway.

**Comment by rm-bergmann** [comment]
Another update, hope this helps. 

Here's 2 screenshots. Codeintel definitely still picks up something, but is limited to 2 classes. If I select it, it inserts the complete heirarchy. I would say it's definitley a bug as it worked corrcetly before.

Thanks

![screen-1](https://user-images.githubusercontent.com/7300540/33141965-bf379572-cfac-11e7-99e7-641cfe0da953.png)

![screen-2](https://user-images.githubusercontent.com/7300540/33141976-c92c9456-cfac-11e7-937d-3b840c6a35da.png)

**Comment by mitchell-as** [comment]
Hi, we changed our underlying CodeIntel architecture in Komodo 11. It's certainly quite possible it "worked" in K10, but perhaps was an unintended consequence.
